### PR TITLE
mkl: 2019.0.117 -> 2019.3.199

### PR DESCRIPTION
**Pull request by markuskowa**
###### Motivation for this change

CC @bhipple 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by markuskowa** [comment]
@GrahamcOfBorg build mkl

File: pkgs/development/libraries/science/math/mkl/default.nix [hunk 1]
```diff
@@ -60,8 +60,8 @@ stdenvNoCC.mkDerivation rec {
   outputHashAlgo = "sha256";
   outputHashMode = "recursive";
   outputHash = if stdenvNoCC.isDarwin
-    then "00d49ls9vcjan1ngq2wx2q4p6lnm05zwh67hsmj7bnq43ykrfibw"
-    else "1amagcaan0hk3x9v7gg03gkw02n066v4kmjb32yyzsy5rfrivb1a";
+    then "00d49ls9vcjan1ngq2wx2q4p6lnm05zwh67hsmj7bnq44ykrfibw"
```
Comment (bhipple) [inline pkgs/development/libraries/science/math/mkl/default.nix]: Last time I did this I changed the hash to all 0's to make it very clear that the Darwin one is busted until someone with a Mac computes it, which was done here: https://github.com/NixOS/nixpkgs/pull/52950

That said, it looks like @smaret is actively using the Mac version (https://github.com/NixOS/nixpkgs/pull/57947) and might be able to compute the Mac hash for this PR.

**Review by bhipple — APPROVED** [review]
LGTM as far as the linux side of things goes.

File: pkgs/development/libraries/science/math/mkl/default.nix (hunk 1 above)
Comment (veprbl) [reply pkgs/development/libraries/science/math/mkl/default.nix]:
```suggestion
    then "0rwm46v9amq2clm6wxhr98zzbafr485dz05pihlqsbrbabmlfw30"
```

Comment (markuskowa) [reply pkgs/development/libraries/science/math/mkl/default.nix]: thanks!

**Pull request merged**